README.md

```diff
@@ -33,9 +33,8 @@ We encourage pull requests and issues tracking via Github, and the [target-devel
 ##### Building tcmu-runner
 
 1. Clone this repo.
-1. Type `./install_dep.sh` to install development packages for dependencies, or you can do it manually:
+1. Type `./extra//install_dep.sh` to install development packages for dependencies, or you can do it manually:
```

Comment:
Thanks for the fixup. I am not very familiar with markdown. Is the extra "/" needed between directories?

Reply:
No, it was a typo. I will fix it. Thanks for the catch.